.vib/phpmyadmin/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/phpmyadmin"
         },
-        "runtime_parameters": "ZGI6CiAgaG9zdDogbWFyaWFkYgogIGFsbG93QXJiaXRyYXJ5U2VydmVyOiB0cnVlCiAgcG9ydDogMzMwNgogIGJ1bmRsZVRlc3REQjogdHJ1ZQpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODAKbWFyaWFkYjoKICBhdXRoOgogICAgcm9vdFBhc3N3b3JkOiByb290UGFzc3dvcmQ=",
+        "runtime_parameters": "aG9zdEFsaWFzZXM6CiAgLSBpcDogMTI3LjAuMC4xCiAgICBob3N0bmFtZXM6CiAgICAgIC0gc3RhdHVzLnRlc3Rob3N0CmNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICBlbmFibGVkOiB0cnVlCiAgcnVuQXNVc2VyOiAxMDAyCnBvZFNlY3VyaXR5Q29udGV4dDoKICBmc0dyb3VwOiAxMDAyCiAgZW5hYmxlZDogdHJ1ZQpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODAKICAgIGh0dHBzOiA0NDQKZGI6CiAgaG9zdDogbWFyaWFkYgogIGFsbG93QXJiaXRyYXJ5U2VydmVyOiB0cnVlCiAgcG9ydDogMzMwNgogIGJ1bmRsZVRlc3REQjogdHJ1ZQptYXJpYWRiOgogIGF1dGg6CiAgICByb290UGFzc3dvcmQ6IHJvb3RQYXNzd29yZA==",
```

Comment:
Are we changing only the HTTP port?

Reply:
Unfortunately, that is a limitation from VIB and the proxy it is behind. Cypress is run in the VIB cluster and accesses PHPMyAdmin through the IP exposed by the LoadBalancer (and the HTTP port), so the proxy limitation is unavoidable.

As we are not accessing the HTTPS port from the outside (goss tests are executed inside the pod), we can work around that limitation for it.